crates/deno_task_shell/src/shell/commands/args.rs

```diff
@@ -26,27 +26,33 @@ impl<'a> ArgKind<'a> {
   }
 }
 
-pub fn parse_arg_kinds(flags: &[String]) -> Vec<ArgKind> {
+pub fn parse_arg_kinds(flags: &mut [String]) -> Vec<ArgKind> {
   let mut result = Vec::new();
   let mut had_dash_dash = false;
-  for arg in flags {
+  let home_dir = dirs::home_dir().unwrap();
```

Comment:
You can combine this line and the one afterwards in one expression. It would also be good to avoid unwrap where possible and prefer to return errors instead. With anyhow and the context method this should be pretty convenient and will give much better error messages.